src/merge_bot/merge_bot.py

```diff
@@ -137,7 +137,7 @@ def message_slack(webhook_url, msg):
 def commit_go_mod_updates(repo):
     try:
         proc = subprocess.run(
-            "go mod tidy", shell=True, check=True, capture_output=True
+            "go mod tidy -compat=1.17", shell=True, check=True, capture_output=True
```

Comment:
Shouldn’t there be a conditional, based on the Go version in go.mod? What if merge_bot is run against an older version that is supposed to compile on a Go v1.16 base image?

Reply:
I wished go was smart enough to use the go version specified in `go.mod` to set the compatibility accordingly during `go mod` :-/